uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/controllers/document-list.js

```diff
@@ -149,9 +161,16 @@ define([
         });
       };
 
+      /**
+       * Creates the date range label using from and until dates in
+       * format 'dd/MM/yyyy'
+       *
+       * @param {string} from
+       * @param {string} until
+       */
       $scope.labelDateRange = function (from, until) {
```

Comment:
Please add test cases to this function since there was a bug here and tests never failed. Please also check if there are another places apart from the controller (https://github.com/compucorp/civihr-tasks-assignments/pull/235/commits/d13d34ce133d08a0bd8b17748da1bbeaaa17d335#diff-c382f8565f58303b3569addc13e28686R136) and if they use from/until parameters. If there are such calls, we need to understand why everything worked before at all.

We may want to test this function with/without params.